frontend/app/src/components/defi/DefiAsset.vue

```diff
@@ -8,7 +8,11 @@
         :value="asset.balance.amount"
         class="defi-asset__amount font-weight-medium"
       />
-      <amount-display :value="asset.balance.usdValue" show-currency="ticker" />
+      <amount-display
+        :value="asset.balance.usdValue"
+        fiat-currency="USD"
+        show-currency="ticker"
```

Comment:
The currency PR has been merged so it would be nice to rebase this PR on top of develop too.

Reply:
Will do!